tests/components/AddonList.test.js

```diff
@@ -120,8 +120,8 @@ describe('<AddonList />', () => {
     });
 
     it('should render div tags', () => {
-        expect(testAddonList.find("div")).to.have.length(6);
-        expect(searchedAddonslist.find("div")).to.have.length(6);
+        expect(testAddonList.find("div")).to.have.length(18);
+        expect(searchedAddonslist.find("div")).to.have.length(18);
     });
 
```

Comment:
Is it possible to write a more comprehensive test for this since everytime a new addon is installed, the test will always fail, the number won't always be 18.

Reply:
The test checks for the number of divs in the file and these can change based on the implementation